Test/Invoke-WhiskeyNodeTask.Tests.ps1

```diff
@@ -391,6 +411,11 @@ function ThenNpmCleanedUp
     }
 }
 
+function ThenNPMIsInstalled {

```

Comment:
Rename this to `ThenNodeModulesAreInstalled`.

Reply:
Done